seismic/analyze_station_orientations.py

```diff
@@ -91,6 +98,5 @@ def main(src_h5_event_file, network=None, station=None, dest_file=None):
 
 
 if __name__ == '__main__':
-    main('/g/data1a/ha3/am7399/shared/7X_RF_analysis/7X_event_waveforms_for_rf_20090616T034200-20110401T231849_rev2.h5',
```

Comment:
An example run is always useful. So please keep the concrete example.

Reply:
Added example usage back in. Note that more cleanup will be done under new ticket PV-276.